Automattic/woocommerce-payments
# chore: remove gateway assignment on init

Fixes #

#### Changes proposed in this Pull Request









This line should not have been there. It looks like I re-added it after a bad merge conflict here: f776f0441a37e6738a2444a50068d7c79b73dadc

The `gateway` is being injected in the constructor, so there's no need for it here.

Removing.

#### Testing instructions





- In WCPay dev, ensure you have the "Enable grouped settings" feature enabled
- Go to the WCPay settings page: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments
- Change some settings, maybe even in the "customize appearance" page: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments&method=digital_wallets
- Settings are still honored by the frontend (ensure you're _not_ using localhost for the frontend, use Jurassic Tube or ngrok instead)

-------------------
